modules/aggregation-layers/src/types.ts

```diff
@@ -1,4 +1,4 @@
-export type AggregateAccessor<DataT = any> = {
+export type AggregateAccessor<DataT = any> = (args: {
```

Comment:
A JSDoc comment before each type would be nice.
```suggestion
/** Type assigned to e.g. getColorValue/getElevationValue props in aggregation layers (e.g. CPUGridLayer, HexagonLayer)*/ 
export type AggregateAccessor<DataT = any> = (args: {
```